cmd/http-content-check/main.go

```diff
@@ -42,26 +44,53 @@ func main() {
 	if TargetString == "" {
 		reportErrorAndStop("No string provided in YAML")
 	}
+}
+
+func main() {
+
+	checkTimeLimit := time.Minute * 1
+	ctx, ctxCancel := context.WithTimeout(context.Background(), checkTimeLimit)
+
+	kubernetesClient, err := kubeClient.Create("")
+	if err != nil {
+		log.Errorln("Error creating kubeClient with error" + err.Error())
+	}
+
+	err = nodeCheck.WaitForKuberhealthy(ctx)
+	if err != nil {
+		ctxCancel()
```

Comment:
we can get rid of this from the suggestion above ^

Reply:
done